framework/packages/abstract-interface/src/native/ans_host.rs

```diff
@@ -142,16 +142,18 @@ impl<Chain: CwEnv> ClientResolve<Chain> for AnsAsset {
     }
 }
 
-impl<Chain: CwEnv> ClientResolve<Chain> for AssetInfo {
```

Comment:
Don't we want this for both AssetInfo and AssetInfoUnchecked?

Reply:
Yes